plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/DeltaLakeQueryRunner.java

```diff
@@ -106,7 +106,7 @@ public DistributedQueryRunner build()
                 if (!deltaProperties.containsKey("hive.metastore.uri")) {
                     Path dataDir = queryRunner.getCoordinator().getBaseDataDir().resolve(DELTA_CATALOG);
                     deltaProperties.put("hive.metastore", "file");
-                    deltaProperties.put("hive.metastore.catalog.dir", dataDir.toString());
+                    deltaProperties.put("hive.metastore.catalog.dir", "file://" + dataDir);
```

Comment:
this is not the correct conversion from path (which eg can contain spaces) to a URI (which eg mustn't contain spaces, using percent-encoding instrad)

Reply:
Changed to `toUri()` instead.